handbook/ops/sales-ops/onboarding.md

```diff
@@ -25,13 +25,13 @@ Start by reading the [operations team onboarding page](../onboarding.md). Once y
 - Read our [customer case studies](https://about.sourcegraph.com/customers)
 - Read our [customer personas](../../marketing/personas.md)
 - Review our sales content and collateral
-  - [Sales content & templates](https://drive.google.com/drive/u/0/folders/1Og99CfsVetXCutZBC01q6Cx54dX5YiyZ)
-  - [ROI calculators](https://drive.google.com/drive/u/0/folders/1dBWklUaRowyj9ZKPU7fKwgPKvsNdfhoy)
-  - [Customer order forms](https://drive.google.com/drive/u/0/folders/1VZGSm7NfcNXv_d2IZ5n6Jl1zwpMoPN7a)
+  - [Sales content & templates](https://drive.google.com/drive/folders/14guSNICPX5bHxUxXIeb1web6MN8z7FA4)
```

Comment:
@joekirscherSG  can you please confirm that all of the replacement links in this folder are correct?